mailstore-connector-test/compose.yml

```diff
@@ -0,0 +1,87 @@
+version: '3'
+
+# ⚠️  SECURITY WARNING: The port configuration below publishes smtp4dev to ALL network interfaces.
+# This means it may be accessible from outside your machine. For security, consider using:
+#   - '127.0.0.1:5000:80'   # Web interface (localhost only)
+#   - '127.0.0.1:25:25'     # SMTP server (localhost only)  
+#   - '127.0.0.1:143:143'   # IMAP server (localhost only)
+# See docs/Docker-Security.md for detailed security guidelines.
+
+services:
+  smtp4dev:
+    image: rnwood/smtp4dev:v3
```

Comment:
with this docker-compose setup I attempted to serve a demo mail-server, since there is nothing in this repo nor any test scenario description.